src/core/resources/WallContentWindow.qml

```diff
@@ -11,6 +11,32 @@ BaseContentWindow {
         anchors.fill: parent
     }
 
+    Rectangle
+    {
+        visible: displaygroup.showWindowTitles
+        y: -Style.windowTitleHeight + ( parent.border.width / 2 )
+        x: 0
+        width: parent.width
+        height: Style.windowTitleHeight - parent.border.width
+        color: parent.border.color
+        border.width: parent.border.width
+        border.color: parent.border.color
+
+        Text
+        {
+            // y delta relative to title rectangle height ( 8 is the best value )
```

Comment:
best value? says who? :-) the line is self-explanatory, just remove this comment :-)